app/services/place_hold_form/builder.rb

```diff
@@ -11,14 +11,7 @@ def initialize(catkey:, user_token:, client:)
 
   def generate
     bib_info = Bib.new(SymphonyClientParser::parsed_response(@client, :get_bib_info, @catkey, @user_token))
-
-    if bib_info.call_list.present?
-      @call_list = bib_info.call_list.map { |call| Call.new record: call }
-      filter_holdables if @call_list.count > 1
-    end
-
-    # no holdable items found
-    return {} if @call_list.blank?
+    return {} unless holdables_present?(bib_info)
```

Comment:
liked this 👌  good to have the generate method shorter as a summary

Reply:
cool